proposals/16-first-filecoin-storage-deal.md

```diff
@@ -0,0 +1,130 @@
+
+#### Impact
+🔥🔥🔥  
+The Filecoin network currently appears to be used primarily by miners, with more supply than demand for storage space, and many docs are focused (explicitly or not) on miner needs with respect to hardware equipment, etc. Developers are unable to quickly and confidently store a file on Filecoin while experimenting with the product, which may lead them to settle for an IPFS pinning service or resort to 3rd party storage solutions. After their first bad experience, they're unlikely to return and try Filecoin again in the future, which makes the situation urgent: we need to either make Filecoin a valid storage solution now for developers or intentionally pause advertising it as one until it meets expectations.
+
+#### Leverage
+🎯 🎯 
+External: Nailing the onboarding path to Filecoin storage is core to promotion of the W3DT stack. If we define the stack in a way that requires including Filecoin, we are unable to educate folks on how to combine pieces of the stack until the onboarding process for Filecoin storage is unblocked. 
+
+Internal:  By necessity, this project will require us to set up a workflow for x-functional communication/coordination between PM, engineering, and documentation/education teams, which can be leveraged for future W3DT work.
+
+
+#### Confidence
+_How sure are we that this impact would be realized? Label from [this scale](https://medium.com/@nimay/inside-product-introduction-to-feature-priority-using-ice-impact-confidence-ease-and-gist-5180434e5b15)_.
+
+**NEED HELP HERE**
+
+Nailing the onboarding process for Filecoin won't guarantee successful adoption, but not nailing it will guarantee failure. 
+
+## Project definition
+#### Brief plan of attack
+
+Step 1: _DONE!_ Have sample users attempt to store a file on Filecoin by walking through existing available documention, with no help from fellow Labbers. 
+
+Step 2: PM creates a system for tracking and assinging the critical tasks on this path, using Andrew's video and [JV's slides](https://docs.google.com/presentation/d/1UbO7LKo47KTZcPL8xK41ubFEZwJ7MeFnnAT0_y1yIII/edit?usp=sharing) as a starting point, particularly slide 42 where he outlines which problems can be solved by documentation and which require engineering (and presumably subsequent changes to documentation).
+
+Step 3 (Docs): Docs writer fixes the documentation pain points by adding more details, ensuring recommendations are tailored to storage clients (not miners), etc. Flags additional pain points found along the way. 
+
+Step 3 (Eng): Engineers tackle identified issues with either core protocol implementation for dealmaking or user experience needs in the CLI (progress indicators, etc.). Flags changes made that will require more updates to docs.
```

Comment:
I think Raul/Dirk’s proposal on deal success interface encompasses a large chunk of this eng work (and should likely be done first)